types/opossum/index.d.ts

```diff
@@ -1,23 +1,26 @@
 // Type definitions for opossum 1.9
 // Project: https://github.com/nodeshift/opossum
 // Definitions by: Quinn Langille <https://github.com/quinnlangille>
+//                 Willy Zhang <https://github.com/merufm>
 //                 Lance Ball <https://github.com/lance>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 /// <reference types="node"/>
 import * as stream from "stream";
+import { EventEmitter } from "events";
 
-export type Action = () => any;
+export type Action = (...args: any[]) => any;
 
-export class CircuitBreaker {
+export class CircuitBreaker extends EventEmitter {
     clearCache(): void;
     close(): void;
     disable(): void;
     enable(): void;
-    fallback(): CircuitBreaker;
-    fire(): Promise<any>;
-    healthCheck(func: () => Promise<any>, interval: number): Promise<any>;
+    fallback(func?: (...args: any[]) => any): this;
+    fire(...args: any[]): Promise<any>;
```

Comment:
Based on the documentation `fire` doesn't seem to received any parameters. Could you confirm if this is ok?

Reply:
@scttcper @armanio123 the PR has merged, but as the author of this (and opossum), this is correct. `fire` receives any arguments and proxies them.